client/signup/jetpack-connect/plans-landing.jsx

```diff
@@ -16,28 +16,28 @@ import QueryPlans from 'components/data/query-plans';
 
 const CALYPSO_JETPACK_CONNECT = '/jetpack/connect';
 
-const PlansLanding = React.createClass( {
+class PlansLanding extends Component {
 
```

Comment:
I'm starting to think you're testing my eyesight :rofl: Or is that part of a grand plan to get this into Calypso's standards? 🤔  😆